core/workspace.js

```diff
@@ -607,6 +629,35 @@ Blockly.Workspace.prototype.getBlockById = function(id) {
   return this.blockDB_[id] || null;
 };
 
+/**
+ * Find the comment on this workspace with the specified ID.
+ * @param {string} id ID of comment to find.
+ * @return {Blockly.WorkspaceComment} The sought after comment or null if not
+ *     found.
```

Comment:
nit: extra tab by the looks of it

Reply:
Line-wrapped block tags are indented by four spaces: https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping